### PR TITLE
add danish lang

### PR DESCRIPTION
**Pull request by NoahBohme**
## Checklist

Please put "X" in the below checkboxes that apply:

- [x] If committing a change of production code, I have tested it in different browsers (Chrome, Firefox, Safari). 
- [x] If committing a new feature, I have first submitted an issue (Please note: you are free to open PRs for non-issued features, but opening an issue increases your chance of a successful PR). 
- [x] If committing a new feature, I have also written the appropriate tests for it. 
- [x] I have tried to build the feature in a way, that it does not cause any breaking changes for others (unless 
  agreed upon in an issue). 

## This PR solves the following problem**. 

Add danish languge

### CODE REVIEW

File: packages/translations/src/locales/da-DK/calendar.ts [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import { CalendarTranslations } from '../../types/calendar.translations'
+
+export const calendarSvSE: CalendarTranslations = {
```
Comment (tomosterlund) [inline packages/translations/src/locales/da-DK/calendar.ts:3]:
```suggestion
export const calendarDaDK: CalendarTranslations = {
```

File: packages/translations/src/locales/da-DK/date-picker.ts [hunk 2]
```diff
@@ -0,0 +1,10 @@
+import { DatePickerTranslations } from '../../types/date-picker.translations'
+
+export const datePickerSvSE: DatePickerTranslations = {
```
Comment (tomosterlund) [inline packages/translations/src/locales/da-DK/date-picker.ts:3]:
```suggestion
export const datePickerDaDK: DatePickerTranslations = {
```

File: packages/translations/src/locales/da-DK/index.ts [hunk 3]
```diff
@@ -0,0 +1,8 @@
+import { datePickerSvSE } from './date-picker'
+import { Language } from '../../types/language.translations'
+import { calendarSvSE } from './calendar'
+
+export const svSE: Language = {
```
Comment (tomosterlund) [inline packages/translations/src/locales/da-DK/index.ts:5]:
```suggestion
export const daDK: Language = {
```

**Review by tomosterlund — CHANGES_REQUESTED** [review]
Hey there. Thanks for wanting to contribute to the library! Just a few minor corrections, and we'd also need to add the full language from `packages/translations/src/locales/da-DK/index.ts` in `packages/translations/src/index.ts` where all translations come together.

Let me know if you need any help, or want me to finish it off :)

**Comment by NoahBohme** [comment]
Thanks - sorry for the delay.

**Comment by tomosterlund** [comment]
No worries! Thanks for adding the translations :)

**Pull request merged**